misc/expansions/BillNoface.class.php

```diff
@@ -10,7 +10,15 @@ public function __construct($row = null)
 
     ];
     $this->bullets = 4;
-    $this->expansion = DODGE_CITY;  
+    $this->expansion = DODGE_CITY;
     parent::__construct($row);
   }
+
+  /**
+   * @return int
+   */
+  public function defaultCardsToDraw()
+  {
+      return $this->bullets - $this->hp + 1;
```

Comment:
Is my assumption correct? bullets = max life points, hp = actual life points?

Reply:
Yes, this is correct